app/src/main/java/com/zulip/android/util/ListDialog.java

```diff
@@ -48,7 +51,8 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
 
         // Inflate and set the layout for the dialog
         // Pass null as the parent view because its going in the dialog layout
-        View rootView = inflater.inflate(R.layout.list_dialog, null);
+        final ViewGroup nullParent = null;
+        View rootView = inflater.inflate(R.layout.list_dialog, nullParent);
```

Comment:
I see, how about just suppressing this error since we are using `null` again as the view root here.